.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,8 +13,8 @@
 - take over the world
 
 #### Fixes the following issue(s)
-<!-- Also add any other links relevant to your change. -->
-- 
+<!-- Prefix issues by "Fixes" so that GitHub links them. Also add any other relevant links. -->
+- Fixes #
```

Comment:
```suggestion
<!-- Prefix issues with "Fixes" so that GitHub closes then when the PR is merged. Also add any other relevant links.

Example:
- Fixes #1
- Fixes #2
- example.com
-->
-
```

I'm not sure if we should have an item already filled with "Fixes #" or not to be honest.

Reply:
I don't like this, it's too long to skim through each time and uses too much space. I made the comment as short as possible so that people tend to read it more. I think adding "Fixes #" is enough of an example, and it is also useful because one does not have to write it manually each time.

Reply:
The issue is that many people don't realize that "fixes" should only be followed by one issue, not multiple. I'll edit my suggestion in a minute to account for your notes.

Reply:
Ok, that looks good